packages/tocco-ui/src/Icon/Icon.js

```diff
@@ -76,7 +80,13 @@ Icon.propTypes = {
   /**
    * Add spacing according position. Default value is 'none'. Possible values: after|before|between|none
    */
-  position: PropTypes.oneOf(['after', 'before', 'between', 'solely'])
+  position: PropTypes.oneOf(['after', 'before', 'between', 'solely']),
+
+  // TODO desribe props
```

Comment:
Take care of the TODO (or remove the comment - at least to me it's quite obvious what the following props are for)